Modernize look

rbeeger opened the pull request:
I use this extension quite often. Thanks for making it.
I thought it could use a fresher more modern look. So I started playing around with the CSS and came up with something I think looks nice. I also added support for dark mode.

Tested it on macOS and windows 10 with Firefox Developer Edition 79.0b9. There's no fancy new CSS features. It should also work with some older versions of Firefox. Haven't checked how far back you can go until it starts to break, though.

I replaced all size definitions with sizes in px. The mix of px and em/pt looked somewhat confusing.

![PixelSnap 2020-07-19 at 10 13 39](https://user-images.githubusercontent.com/426834/87870516-10fea000-c9a9-11ea-9060-bf0ab3218710.png)
![PixelSnap 2020-07-19 at 10 14 10](https://user-images.githubusercontent.com/426834/87870517-12c86380-c9a9-11ea-9285-e523805f50dc.png)

Cito commented:
Thank you for this contribution @rbeeger - looks much better than my meager CSS.

Will be merged and a new version released as soon as I find some time.

Cito commented:
The new look has now been released in version 0.3.0.

The only thing I added is a visible checkmark in the checkboxes in checked state, to make it clearer which state they are in.

rbeeger commented:
> The new look has now been released in version 0.3.0.
Thanks.

> The only thing I added is a visible checkmark in the checkboxes in checked state, to make it clearer which state they are in.
Yes, it looks better with a checkmark. I was too lazy. The Firefox default didn't match. So I replaced it with the filled or unfilled box. Now there can be no confusion about which state the checkboxes are in 👍
